core/jazz_environment-event-handler/config/prod-config.json

```diff
@@ -54,5 +54,13 @@
     "website": "build-pack-website",
     "sls-app": "build-pack-sls-app",
     "ui": "jazz_ui"
+  },
+  "TVAULT": {
+    "IS_ENABLED": true,
+    "ENDPOINT": "{endpoint_url}",
+    "API": "{t-vault_api}",
+    "CREATE_SAFE_API_RESOURCE": "/jazz/t-vault/safes",
+    "SAFE_API": "/t-vault/safes/",
+    "HOSTNAME": "http://secret1024-tvault-alb-420222041.us-east-1.elb.amazonaws.com"
```

Comment:
Why this HOSTNAME ? Please reuse the existing keys

Reply:
using Hostname from config_loader. changes Updated.